packages/lib/src/toolbar/controls/ExportMenu.tsx

```diff
@@ -0,0 +1,60 @@
+import { isDefined } from '@h5web/shared';
+import { Button, Wrapper, Menu } from 'react-aria-menubutton';
+import { FiDownload } from 'react-icons/fi';
+import { MdArrowDropDown } from 'react-icons/md';
+
+import styles from './Selector/Selector.module.css';
+
+interface Props<F extends string> {
+  formats: F[];
+  isSlice: boolean;
+  getFormatURL: (format: F) => string | undefined; // `undefined` if format is not supported
+}
+
+function ExportMenu<F extends string>(props: Props<F>) {
+  const { formats, isSlice, getFormatURL } = props;
+
+  const urls = getFormatURL ? formats.map(getFormatURL) : [];
```

Comment:
Given that `getFormatURL` is always defined, you can get rid of the ternary

Reply:
Well spotted